scripts/build-json/build-guide-page-json.js

```diff
@@ -9,7 +10,15 @@ async function processDirective(elementPath, directive) {
         case 'embed-example':
             return {
                 type: 'example',
-                value: await examples.packageExample(path.join(elementPath, directiveComponents[1]))
+                value: await packageExample(path.join(elementPath, directiveComponents[1]))
+            }
+        case 'embed-compat':
+            return {
+                type: 'browser_compatibility_table',
```

Comment:
I don't know if I can see the light. Why introduce another type when you *could* just re-use the `browser_compatibility` and simply omit the `title` and `id` as a hint to the renderer. Or, perhaps I just misread your future thinking in the PR description. 

You suggested this diff:
```diff
-        <h2 id="Browser_compatibility">Browser Compatibility</h2>
+        {data.title && <h2 id={data.id}>{data.title}</h2>}
```
which would satisfy and be able to use just 1 type enum/string for these sections.